poetry.lock

```diff
@@ -535,8 +535,8 @@ attrs = "*"
 importlib-metadata = {version = ">=1.4", markers = "python_version < \"3.8\""}
 
 [package.extras]
-test = ["aiounittest", "twisted", "tox"]
-dev = ["twine (==4.0.1)", "build (==0.8.0)", "isort (==5.9.3)", "flake8 (==4.0.1)", "black (==22.3.0)", "mypy (==0.910)", "aiounittest", "twisted", "tox"]
+dev = ["tox", "twisted", "aiounittest", "mypy (==0.910)", "black (==22.3.0)", "flake8 (==4.0.1)", "isort (==5.9.3)", "build (==0.8.0)", "twine (==4.0.1)"]
```

Comment:
Not specific to this PR, but what's with the version of mypy here being stuck at 0.910?

Reply:
This is a copy of the requirements declared by matrix-common. Its `dev` extra pins a specific version of mypy here: https://github.com/matrix-org/matrix-python-common/blob/b1801eed82b1a9c79cfbc8f85a8c79d11855f48d/setup.cfg#L32

Poetry ignores this constraint though, because Synapse depends on `matrix-common`, not `matrix-common[dev]`.